modules/phase_field/tutorials/spinodal_decomposition/s1_testmodel.i

```diff
@@ -0,0 +1,147 @@
+#
+# Simulation of an iron-chromium alloy using simplest possible code and a test
+# set of initial conditions.
+#
+
+[Mesh]
+  # generate a 2D, 25nm x 25nm mesh
+  type = GeneratedMesh
+  dim = 2
+  elem_type = QUAD4
+  nx = 100
+  ny = 100
+  nz = 0
+  xmin = 0
+  xmax = 25
+  ymin = 0
+  ymax = 25
+  zmin = 0
+  zmax = 0
+[]
+
+[Variables]
+  [./c]   # Mole fraction of Cr (unitless)
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./w]   # Chemical potential (eV/mol)
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[ICs]
+  # Use a bounding box IC at equilibrium concentrations to make sure the
+  # model behaves as expected.
+  [./testIC]
+    type = BoundingBoxIC
+    variable = c
+    x1 = 5
+    x2 = 20
+    y1 = 5
+    y2 = 20
+    inside = 0.823
+    outside = 0.236
+  [../]
+[]
+
+[BCs]
+  # periodic BC as is usually done on phase-field models
+  [./Periodic]
+    [./c_bcs]
+      variable = c
```

Comment:
If you leave the variable line out, it will automatically apply the BC to every variable so you will only need one block.